src/skills-builder/skills-builder-modal/select-preferences/SelectPreferences.jsx

```diff
@@ -12,26 +11,24 @@ import messages from './messages';
 
 const SelectPreferences = () => {
   const { formatMessage } = useIntl();
-  const { state } = useContext(SkillsBuilderContext);
-  const { currentGoal, currentJobTitle } = state;
```

Comment:
So we don't need to use State at all anymore in this component? Remind me -- we were using the state to track/manage what "step" the learner is currently in? So now that we are displaying all the questions at once, we don't need to track the state?

Reply:
Yes exactly. We don't need to hold the state because the UI will only be set at the beginning and not modified throughout the user experience. The `useRef` hook will just extract the values once and not have any effect on re-renders